examples/example3/README.md

```diff
@@ -0,0 +1,32 @@
+
+# Example 3
+
+This is a simple example to show how DiffSync can be used to compare and synchronize data with a remote system like via a REST API like Nautobot.
+
+For this example, we have a shared model for Region and Country defined in `models.py`.
+A Country must be associated with a Region and can be part of a Subregion too.
+
+The comparison and synchronization of dataset is done between a local JSON file and the [public instance of Nautobot](https://demo.nautobot.com).
+
+
+## Install the requirements
+
+to use this example you must have some dependencies installed, please make sure to run 
+```
+pip install -r requirements.txt
+```
+
+## Try the example
+
+The first time a lot of changes should be reported between Nautobot and the local data because by default the demo instance doesn't have the subregion define.
```

Comment:
```suggestion
The first time you run this example, a lot of changes should be reported between Nautobot and the local data because by default the demo instance doesn't have the subregion defined.
```

Would be good to include a snippet of sample output here as well.